### DISCUSSION TITLE
Outreachy Introduction

### DISCUSSION THREAD

**Discussion by kleenpulse** [ModECI/MDF#511] [General]
Hello all,
I'm Isaac Gideon a Front-end web dev also an Outreachy internship applicant!

**Comment by AfnanHussain10** [comment]
Greetings, everyone!

I'm Afnan Hussain, a passionate data science undergraduate eagerly seeking an opportunity to make meaningful contributions. Currently applying for the Outreachy internship, I'm excited about the prospect of leveraging my skills and knowledge to enhance this project. I look forward to collaborating and making a positive impact.

**Comment by ektuu** [comment]
Hi everyone!
I’m Ekta and I’m really excited to get involved in open source communities and projects.
I’m a second year undergrad from India pursuing Computer science as my majors.

I’ve been indulged in Machine learning since last year, and I came across this project and it really aligns with my interests.

Hope to have lots of learning!

**Comment by pgleeson** [comment]
Hi all! Please see general advice on this round of applications for Outreachy here: https://github.com/ModECI/MDF/discussions/509
